pkg/cloudprovider/suite_test.go

```diff
@@ -146,8 +147,9 @@ var _ = BeforeSuite(func() {
 	recorder = coretest.NewEventRecorder()
 	prov = provisioning.NewProvisioner(ctx, env.Client, env.KubernetesInterface.CoreV1(), recorder, cloudProvider, cluster)
 	provisioningController = provisioning.NewController(env.Client, prov, recorder)
+	nodeTemplateController = nodetemplatestatus.NewController(env.Client, fakeEC2API, subnetCache, securityGroupCache)
 
-	env.CRDDirectoryPaths = append(env.CRDDirectoryPaths, RelativeToRoot("charts/karpenter/crds"))
+	env.CRDDirectoryPaths = append(env.CRDDirectoryPaths, utils.RelativeToRoot("charts/karpenter/crds"))
```

Comment:
I don't think this line is needed anymore now that we have CRDs passed in the environment startup. Let's check if we can delete this